src/bootstrap-tenant/bootstrapper/index.ts

```diff
@@ -466,10 +475,7 @@ export class Bootstrapper extends EventEmitter {
 
       // Shapes
       if (props.shapes) {
-        spec.shapes = await getExistingShapesForSpec(
-          this.context,
-          props.onUpdate
-        )
+        spec.shapes = await getExistingShapesForSpec(this.context)
```

Comment:
No progress feedback here then? Do we eventually mark the shape progress as 100%?

Reply:
The `onUpdate` passed to `getExistingShapesForSpec` looks like it was only used for emitting errors to from calling the `validShapeIdentifier` function. This validation is now handled by the `import-export-sdk`/`schema`, and this error is caught when the import-export-sdk is called, so the error should be emitted, just in a different place